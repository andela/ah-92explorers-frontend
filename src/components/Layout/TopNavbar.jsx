import React, { Component, Fragment } from 'react';
import { Link } from 'react-router-dom';
import logo from '../../assets/icons/logo.png';

class TopNavbar extends Component {
  render() {
    return (
      <Fragment>
        <div className="navbar">
          <img src={logo} alt="" className="logo dash-logo" />
          <div className="restOfNav">
            <ul>
              <li>
                {' '}
                <Link to="/login">Login</Link>
              </li>
              <li>
                {' '}
                <Link to="/signup">Sign up</Link>
              </li>
            </ul>
          </div>
        </div>

      </Fragment>
    );
  }
}

export default TopNavbar;
