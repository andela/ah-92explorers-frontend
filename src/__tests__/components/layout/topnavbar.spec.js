/* eslint-disable import/no-unresolved */
import React from 'react';
import { shallow } from 'enzyme';
import { Provider } from 'react-redux';
import configureMockStore from 'redux-mock-store';
import TopNavbar from '../../../components/Layout/TopNavbar.jsx';

const mockStore = configureMockStore();
const store = mockStore({});
describe('<TopNavbar />', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(
      <Provider store={store}>
        <TopNavbar />
      </Provider>,
    );
  });

  it('should render TopNavbar component', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
