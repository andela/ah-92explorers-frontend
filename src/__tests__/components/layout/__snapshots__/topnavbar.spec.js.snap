// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TopNavbar /> should render TopNavbar component 1`] = `ShallowWrapper {}`;
